<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesDialog</class>
 <widget class="QDialog" name="FilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Device Setup</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Electric field (V/cm)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Weighting potential</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_efield">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gunn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Efield">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Gunn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobility</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lifetime</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mun">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(cm2/V/s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taun">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mup">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_taup">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_temp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturation Velocity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vsatp">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_vsatn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(K)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Electron</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hole</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(cm/s)</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="gunn_plot" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="efield_plot" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>757</x>
     <y>450</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_efield">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_gunn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_Efield</tabstop>
  <tabstop>pushButton_efield</tabstop>
  <tabstop>lineEdit_Gunn</tabstop>
  <tabstop>pushButton_gunn</tabstop>
  <tabstop>lineEdit_mun</tabstop>
  <tabstop>lineEdit_taun</tabstop>
  <tabstop>lineEdit_mup</tabstop>
  <tabstop>lineEdit_taup</tabstop>
  <tabstop>lineEdit_vsatn</tabstop>
  <tabstop>lineEdit_vsatp</tabstop>
  <tabstop>lineEdit_temp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>FilesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>FilesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>711</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
